packages/pn-pa-webapp/src/utils/constants.ts

```diff
@@ -8,3 +8,11 @@ export const API_BASE_URL = process.env.REACT_APP_URL_API;
 export const SELFCARE_URL_FE_LOGIN = process.env.REACT_APP_URL_SELFCARE_LOGIN;
 export const PAGOPA_HELP_EMAIL = process.env.REACT_APP_PAGOPA_HELP_EMAIL;
 export const ENABLE_ASSISTANCE = process.env.REACT_APP_ENABLE_ASSISTANCE === 'true';
+
+const sUsrAg = navigator.userAgent;
+export const IS_CHROME = sUsrAg.indexOf("Chrome") > -1;
+export const IS_FIREFOX = sUsrAg.indexOf("Firefox") > -1;
+export const IS_EXPLORER = sUsrAg.indexOf("Trident") > -1;
+export const IS_EDGE = sUsrAg.indexOf("Edge") > -1;
+export const IS_SAFARI = sUsrAg.indexOf("Safari") > -1;
+export const IS_OPERA = sUsrAg.indexOf("Opera") > -1 || sUsrAg.indexOf("OPR") > -1;
```

Comment:
in base ai commenti precedenti, valutare se queste costanti sono ancora necessarie, qual ora lo fossero rinominare `sUsrAg `in `currentUserAgent`